PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -266,7 +265,7 @@ def miniAOD_customizeCommon(process):
         src = 'ak4PFJetsPuppi',
         jetCorrLabel = 'ak4PFCHSL2L3Corrector',
     )
-    del process.corrPfMetType1Puppi.offsetCorrLabel # no L1 for PUPPI jets
+    # del process.corrPfMetType1Puppi.offsetCorrLabel # no L1 for PUPPI jets
```

Comment:
please remove if not needed;
if needed, either add an option to pick it up or leave a comment when and how someone may want to uncomment this.